common/psbt_open.c

```diff
@@ -491,3 +491,16 @@ bool psbt_has_our_input(const struct wally_psbt *psbt)
 
 	return false;
 }
+
+bool psbt_contribs_changed(const tal_t *ctx,
+			   struct wally_psbt *orig,
+			   struct wally_psbt *new)
+{
```

Comment:
This is an antipattern!  It doesn't return an allocated object, so why pass in a ctx?

I think you meant to use tmpctx internally.

Reply:
ACK, yes i can do that. Since this is common it seems .. better? to use an internal allocation/free cycle -- is that the wrong intuition?

Reply:
Hmm, tmpctx is pretty universal now.  Either way.